.travis.yml

```diff
@@ -1,7 +1,11 @@
 language: node_js
 
 node_js:
-  - '4.2.1'
+  - '8'
+
+cache:
```

Comment:
I've seen cause build flake. I'm not saying you  should not do this, but like my previous comment, it could be beneficial to include it in the message. Folks that are trying to diagnose what the hell happened, somewhere in the future, might thank you.